pkg/infra/aws/bucket/aws_test.go

```diff
@@ -62,30 +63,47 @@ func TestS3Client(t *testing.T) {
 	}
 	awsConfig = awsConfig.WithCredentials(credentials.NewSharedCredentials("", ""))
 
-	bucket := "kurator-lacal-test"
+	bucket := "kurator-local-test"
 	client, err := NewS3Client(awsConfig, bucket)
 	assert.NoError(t, err)
 	assert.NotNil(t, client)
 
 	exist := client.BucketExists()
-	assert.Equal(t, exist, false)
+	assert.Equal(t, false, exist)
 
 	err = client.MakeBucket()
 	assert.NoError(t, err)
 
 	err = client.PutObject(&File{
-		Filename: "test.json",
+		Filename: "public.json",
 		Buffer:   bytes.NewBufferString("123456"),
 		ACL:      "public-read",
 	})
 	assert.NoError(t, err)
 
+	resp, err := http.Get("https://s3.us-east-2.amazonaws.com/kurator-local-test/public.json")
```

Comment:
how could this work? what is `kurator-local-test`

Reply:
it's created in L66 and deleted in L104

this test reqired credential in local machine, guard by build tag, will not run on CI